File: Tutorials/30 Days of Code/30-review-loop.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner in = new Scanner (System.in);
        int T = in.nextInt();
        in.nextLine();
        for ( int t = 0; t < T; ++t ) {
            String S = in.nextLine();
            char[] s = S.toCharArray();
            for ( int start = 0; start < 2; ++start ) {
                for ( int i = start; i < s.length; i += 2 )
                    System.out.print( s[i] );
                System.out.print( " " );
            }
            System.out.println();
        }
    }
}

File: Tutorials/30 Days of Code/30-review-loop.cxx
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int T; cin >> T;
    string dummy; getline(cin, dummy);
    for ( int t = 0; t < T; ++t ) {
        string s; getline(cin, s);
        for ( int start = 0; start < 2; ++start ) {
            for ( int i = start; i < s.length(); i += 2 )
                cout << s[i];
            cout << ( start == 0 ? ' ' : '\n' );
        }
    }
    return 0;
}
